src/hotspot/cpu/riscv/nativeInst_riscv.cpp

```diff
@@ -46,44 +46,12 @@ bool NativeInstruction::is_call_at(address addr) {
 }
 
 //-----------------------------------------------------------------------------
-// NativeFarCall
+// NativeCall
 //
 // Implements direct far calling loading an address from the stub section version of reloc call.
```

Comment:
Suggestion: `// Implements indirect far call loading an address from the stub section of reloc call.`

And I think this comment should be moved to immediately before definition of `MacroAssembler::reloc_call` [1].

[1] https://github.com/openjdk/jdk/blob/master/src/hotspot/cpu/riscv/macroAssembler_riscv.cpp#L4982